server/rest/instance.py

```diff
@@ -156,12 +157,18 @@ def updateInstance(self, instance):
     @autoDescribeRoute(
         Description('Delete an existing instance.')
         .modelParam('id', model='instance', plugin='wholetale', level=AccessType.WRITE)
+        .param('force', 'If true, remove object directly rather than via event handler.',
+               default=False, required=False, dataType='boolean')
         .errorResponse('ID was invalid.')
         .errorResponse('Write access was denied for the instance.', 403)
     )
-    def deleteInstance(self, instance, params):
-        self.model('instance', 'wholetale').deleteInstance(
-            instance, self.getCurrentToken())
+    def deleteInstance(self, instance, force):
+        if force:
+            self.model('instance', 'wholetale').remove(instance)
```

Comment:
I may be overthinking things here, but don't we still want to try to call the handler to cleanup the underlying container when forcing? Unless `self.model('instance', 'wholetale').remove(instance)` would somehow trigger an event to clean things up or something..

The description is explicit about what happens to the Instance resource, but it is not clear about what happens to the container itself.